packages/ember-glimmer/tests/integration/helpers/text-area-test.js

```diff
@@ -0,0 +1,58 @@
+import { set } from 'ember-metal/property_set';
+import TextArea from 'ember-views/views/text_area';
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+
+class TextAreaRenderingTest extends RenderingTest {
+  constructor() {
+    super();
+
+    this.registerComponent('-text-area', { ComponentClass: TextArea });
+    this.render('{{textarea disabled=disabled value=val}}', { val: 'Lorem ipsum dolor' });
```

Comment:
Since this is so easy to do now, I think we prefer to inline this into the tests now. (See below)